contracts/TellorFlex.sol

```diff
@@ -153,6 +154,9 @@ contract TellorFlex {
             require(token.transferFrom(msg.sender, address(this), _amount));
         }
         _updateStakeAndPayRewards(msg.sender, _staker.stakedBalance + _amount);
+        if (_staker.startDate == 0) {
```

Comment:
if someone stakes, then completely unstakes, then stakes again with the same address, this logic won't increase the `totalStakers`